app/views/waste_exemptions_engine/contact_address_manual_forms/new.html.erb

```diff
@@ -19,7 +19,9 @@
       <%= link_to(t(".postcode_change_link"), back_contact_address_manual_forms_path(@contact_address_manual_form.token)) %>
     </div>
 
-    <%= render("waste_exemptions_engine/shared/manual_address", form: @contact_address_manual_form, f: f) %>
+    <%= f.fields_for :contact_address do |f| %>
```

Comment:
This allow Rails to generate inputs with the correct name, so that the parameters sent via form submit will arrive in the correct nested state.